lib/js_of_ocaml/typed_array.ml

```diff
@@ -20,7 +20,9 @@
 open! Import
 open Js
 
-type uint32 = float
+type int32 = number_t
```

Comment:
I wonder if we should have a different name to avoid confusion with the predefined `int32` type ? would `int32_` be enough ?

Reply:
Why not. But in fact, I find the two type synonyms rather confusing and have removed them in the last version. What do you think?